ui/apps/platform/src/Containers/NetworkGraph/deployment/DeploymentFlows.css

```diff
@@ -5,3 +5,7 @@
 tbody tr.pf-c-table__expandable-row td.pf-c-table__check label {
     padding: 0;
 }
+
+.advanced-flows-filters-select .pf-c-select__menu {
```

Comment:
If I don't set a min-width, the dropdown is too narrow and the ports select doesn't fit so nicely